package.json

```diff
@@ -154,6 +154,7 @@
     "flow-bin": "0.122.0",
     "imports-loader": "0.7.1",
     "jetifier": "1.6.4",
+    "jsonlint": "^1.6.3",
```

Comment:
Please pin the dependency to the exact version.

Reply:
Done